tests/default_vars.sh

```diff
@@ -782,6 +784,7 @@ export SYEAR=2021
 export SMONTH=03
 export SDAY=22
 export SHOUR=06
+export CHOUR=06
```

Comment:
Why do we need ``chour`` here and in the diag_tables?

Reply:
Because the diag table starting time is different for IAU test.

Reply:
I see, it was already present in the test, which is why I didn't see the assigment. 

What is the "c" in "chour" supposed to mean? For shour, that is "starting hour". So is chour the "continue hour"?

Reply:
Cycle hour. In IAU case, the cycle time is 2021032212, even though the model starts with 2021032206 restart files, the forecast is for 2021032212.